i18n/de_DE.po

```diff
@@ -103,55 +98,60 @@ msgstr "+49..."
 msgid "+41..."
 msgstr "+49..."
 
-#: app/scripts/app.js:132
+#: app/scripts/app.js:134
 msgid "-"
 msgstr ""
 
 #: app/scripts/produzenten/detail/produzentendetail.html:99
 msgid "2.0"
 msgstr "2.0"
 
+#: app/scripts/kunden/detail/kundendetail.html:361
+msgid "46.947480"
+msgstr ""
+
+#: app/scripts/kunden/detail/kundendetail.html:364
+msgid "7.458070"
+msgstr ""
+
 #: app/scripts/arbeitsangebote/detail/arbeitsangebotedetail.html:75
 msgid "<button ng-if=\"!arbeitsangebotForm.$invalid\" type=\"button\" role=\"button\" ng-click=\"aaMultiplizieren()\" class=\"btn btn-default\"><span class=\"glyphicon glyphicon-plus\"></span> <span translate=\"\">Multiplizieren</span></button>"
 msgstr ""
 
-#: app/scripts/abos/detail/abosdetail.html:189
-msgid "<input id=\"zusatzaboPrice\" class=\"col-md-9\" type=\"number\" step=\"0.01\" ng-model=\"zusatzAbo.price\">\n"
-"                    <div ng-if=\"zusatzAbo.price != zusatzAbo.zusatzaboTypPrice\" class=\"col-md-1\"><button type=\"button\" class=\"btn btn-xs\" ng-click=\"defaultZusatzaboPrice(zusatzAbo.id)\" title=\"{{'Den Standardpreis wiederherstellen'|translate}}\"><span class=\"glyphicon glyphicon-erase\"></span> </button>\n"
-"                    </div>"
+#: app/scripts/reports/detail/reportsdetail.html:14
```

Comment:
This looks suspicious as there isn't a template change in addition. On purpose?

Also, ability to have a different icon per language on purpose?

Reply:
done